src/mock/util/spa-mock-utils.tsx

```diff
@@ -31,6 +25,11 @@ export const dialogSpaMockProps: SpaMockProps = {
 	tekst: 'Dialog',
 	className: 'spa-mock__content--dialog'
 };
+export const overblikkSpaMockProps: SpaMockProps = {
```

Comment:
Tror ikke vi får noe ut av bare å legge til overblikk her, fordi den importeres ikke som en NavSpa-app, men som en web-component! Tror i så fall det må mer mocking til for å få ut noe synlig 🤔

Reply:
Fjernet overblikkSpaMockProps. Vi får se på mocking av en Overblikk-side seinere.